packages/sdk/src/utils.ts

```diff
@@ -15,3 +16,22 @@ export function getFeeOracleBaseURL(environment?: Environment): string {
       return MAINNET_FEE_ORACLE_BASE_URL;
   }
 }
+
+export async function getTransferStatus(
```

Comment:
I would add docs to this function, explaining that txHash is deposit transaction txHash